midealocal/device.py

```diff
@@ -593,93 +576,114 @@ def _check_heartbeat(self, now: float) -> None:
             self.send_heartbeat()
             self._previous_heartbeat = now
 
-    def run(self) -> None:
+    def _connect_loop(self) -> None:
+        """Connect loop until device online."""
+        # connect loop until online
```

Comment:
one device only have one thread and it will sleep for 10 minutes. there's no more addition resource cost.

if we can fully control with it, why we select raise error to HA? and this is out of control and also may introduce new issue.
no one test it and no one can confirm with it.

and this code process has been test and confirmed in too many release.

before we got a fully test and confirm it can fully works. rollback to a known solution is the best choice for us now.

these bug exist for so many years.
user always rollback to v0.5.4 version and they have many suggestion for us,and also confirm with us that v0.5.4 should the best option and they have test with all the new release.